test/functional/sync_plan/update_test.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../test_helper'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+
+describe 'update a sync plan' do
+  include OrganizationHelpers
+  include SyncPlanHelpers
+
+  let(:org_id) { 1 }
+  let(:id) { 1 }
+  let(:name) { "sync_plan1" }
+  let(:org_name) { "org1" }
+  let(:desc) { "New Description" }
+
+  it 'with organization id and sync plan ID' do
```

Comment:
This is a bit weird, but I'll accept it. :) The `--organization-id` isn't really useful with `--id`. It's only used to look up the ID using the sync plan name, but I guess it could make sense to test that organization ID *isn't* being sent to the API, which is one thing this test does.